src/app/pages/HomePage/HomePage.tsx

```diff
@@ -59,7 +59,7 @@ const HomePage = ({ pageData }: HomePageProps) => {
         entities={[itemList]}
       />
       <main css={styles.main}>
-        <ATIAnalytics atiData={{ ...metadata, title }} />
+        <ATIAnalytics atiData={metadata} />
```

Comment:
This is a weee bit confusing for me - I would rather that we explicitly pass `metadata.atiAnalytics` here... or rename the `metadata` prop above to `atiAnalytics`?